WordPress/gutenberg
# Site Editor: Fix Invisible Template Previews in the Sidebar

## Description

A recent change on the FSE navigation sidebar's overflow accidentally hid the templates (and template parts) previews that were supposed to appear when hovering on the sidebar items.

This PR moves the preview SlotFill outside of the sidebar hidden overflow, and reposition it accordingly.

## How has this been tested?

- Open the Site Editor.
- Open the navigation sidebar.
- Enter either the Templates or Template Parts menu.
- Move the mouse over a template (or part).
- Make sure the template preview is visible, just below the header, and doesn't have z-index issues.

## Screenshot

<img width="619" alt="Screenshot 2020-10-23 at 16 28 56" src="https://user-images.githubusercontent.com/2070010/97023104-ef372b00-154c-11eb-8596-2ace48e47435.png">


## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
